pandas/tests/indexes/test_datetimelike.py

```diff
@@ -867,7 +867,7 @@ def test_repeat(self):
         # GH10183
         idx = pd.period_range('2000-01-01', periods=3, freq='D')
         res = idx.repeat(3)
-        exp = PeriodIndex(idx.values.repeat(3), freq='D')
+        exp = PeriodIndex._simple_new(idx.values.repeat(3), freq='D')
```

Comment:
OK - I can have `.from_ordinals` point directly to `._simple_new`, or were you thinking something else?